assets/queries/k8s/streaming_connection_timeout_disabled/metadata.json

```
@@ -0,0 +1,10 @@
+{
+  "id": "ed89b97d-04e9-4fd4-919f-ee5b27e555e9",
+  "queryName": "Streaming Connection Timeout Disabled",
```

```suggestion
       "queryName": "Kubelet Streaming Connection Timeout Disabled",
```

Please, also update the query folder name and the `positive_expected_result.json`